content/src/fundings/angel-match-program.md

```diff
@@ -32,23 +32,24 @@ sector:
   - other-services
   - technology
 publishStageArchive: ""
-descriptionMd: The Angel Match Program provides funding from the State Small
+sidebarCardBodyText: The Angel Match Program provides funding from the State Small
   Business Credit Initiative (SSBCI) by matching angel investors’ direct
   investment in early-stage, product-based technology companies within targeted
   industries.
 ---
+
 ## Eligibility
 
-* Must have minimum revenues of $100,000 within the `trailing 12 months|trailing-12-months`. Trailing 12 Months (TTM) figures report metrics based on the last 12 months (or four quarters) to date on a rolling basis.
-* Business must be able to demonstrate a minimum of $100,000 in investment interest from at least two investors independent from the company
-* Primary business is commercializing and marketing a product
-* Early stage within an [NJEDA targeted industry](https://www.njeda.gov/wp-content/uploads/2022/11/Appendix-C-Targeted-Industries-Definitions-12.6.22_v2.pdf)
-* Structured as a C-Corporation or LLC
-* Located in physical commercial, co-working, or incubator space in New Jersey
-* A minimum of two full-time founders/C-level executives and 50% of full-time employees working in New Jersey
-* No more than 100 total employees working for the company
-* Company must be registered in New Jersey and in good standing as certified by a valid NJ Tax Clearance Certificate
+- Must have minimum revenues of $100,000 within the `trailing 12 months|trailing-12-months`. Trailing 12 Months (TTM) figures report metrics based on the last 12 months (or four quarters) to date on a rolling basis.
+- Business must be able to demonstrate a minimum of $100,000 in investment interest from at least two investors independent from the company
+- Primary business is commercializing and marketing a product
+- Early stage within an [NJEDA targeted industry](https://www.njeda.gov/wp-content/uploads/2022/11/Appendix-C-Targeted-Industries-Definitions-12.6.22_v2.pdf)
+- Structured as a C-Corporation or LLC
+- Located in physical commercial, co-working, or incubator space in New Jersey
+- A minimum of two full-time founders/C-level executives and 50% of full-time employees working in New Jersey
+- No more than 100 total employees working for the company
+- Company must be registered in New Jersey and in good standing as certified by a valid NJ Tax Clearance Certificate
```

Comment:
I'd like to know what's setting the bullets as `*` and what's setting them as `-`. This is such a minor nit, and it literally doesn't matter because they render the same. I've just seen this multiple times and it makes me wonder. I'm assuming Netlify is doing `*` and all of our editors rewrite it. 

Just thinking out loud.

Reply:
I'm not 100% sure what's happening